README.md

```diff
@@ -1,15 +1,11 @@
-## Tendermint-build container
-Containerized build environemnt to create the docker artifact: 
-https://hub.docker.com/r/iov1/tendermint
-
-### Development
-
-```
-docker build --pull --no-cache -t v0.29.1 --build-arg source_version=v0.29.1 --build-arg source_repo=github.com/tendermint/tendermint .
-```
-
-* ` --no-cache` to not cache the code during checkout step
-
+## Tendermint-build
+[Gitian](https://gitian.org) base build process for [Tendermint](https://github.com/tendermint/tendermint).
+
+#### Trigger new artifact build
+1. Update `./build-version.txt` file and set proper tag and commit hash.
+2. Do PR and let travisCI build it.
+3. Merge to master and Tag with `v0.31.5-iov1`. The first part is the Tendermint tag you are building and the second a iov counter that you can increase if you have to.
```

Comment:
I understand that `v0.31.5-iov1` is a string build using the version information. Maybe it is better to write it

> Merge the Pull Request to `master` branch and create a tag `BUILD_TAG-iovCOUNTER`, (for example `v0.31.5-iov1`)
>  `BUILD_TAG` is the Tendermint tag you are building and the `COUNTER` an IOV counter that you can increase if you have to, to create another build for the same build tag value.